apps/test/unit/templates/sectionProgress/ProgressTableLevelBubbleTest.jsx

```diff
@@ -36,81 +38,87 @@ const assessmentBorders = {
   [LevelStatus.not_tried]: color.lighter_gray,
   [LevelStatus.attempted]: color.level_submitted,
   [LevelStatus.submitted]: color.level_submitted,
-  [LevelStatus.completed_assessment]: color.level_submitted
+  [LevelStatus.completed_assessment]: color.level_submitted,
+  [LevelStatus.perfect]: color.level_submitted
 };
 const assessmentBackgrounds = {
   [LevelStatus.locked]: color.level_not_tried,
   [LevelStatus.not_tried]: color.level_not_tried,
   [LevelStatus.attempted]: color.level_not_tried,
   [LevelStatus.submitted]: color.level_submitted,
-  [LevelStatus.completed_assessment]: color.level_submitted
+  [LevelStatus.completed_assessment]: color.level_submitted,
+  [LevelStatus.perfect]: color.level_submitted
 };
 
 /**
  * Helper function to retrieve the style object of a rendered bubble with the
- * provided status. We use a "bonus" bubble to make it easy to find the
- * FontAwesome content node. The content is in a positioning container, and the
- * parent of that container is the actual "bubble" node with stylized border
- * and background, so we want the style of the content's grandparent.
+ * provided status and prpos.
  */
 function bubbleContainerStyleForStatus(status, propOverrides = {}) {
-  const wrapper = shallow(
+  const wrapper = mount(
     <ProgressTableLevelBubble
       {...defaultProps}
       {...propOverrides}
       levelStatus={status}
-      bonus={true}
     />
   );
+  const bubbleType = propOverrides.concept
+    ? subComps.LargeDiamond
+    : subComps.LargeCircle;
   return wrapper
-    .find(FontAwesome)
+    .find(bubbleType)
     .at(0)
-    .parents()
-    .at(1)
+    .childAt(0)
     .props().style;
 }
 
 describe('ProgressTableLevelBubble', () => {
-  it('renders an anchor tag when enabled', () => {
+  it('renders a link when enabled', () => {
     const wrapper = shallow(<ProgressTableLevelBubble {...defaultProps} />);
-    assert(wrapper.is('a'));
+    expect(wrapper.find(subComps.LinkWrapper)).to.have.lengthOf(1);
   });
 
-  it('does not render an anchor tag when disabled', () => {
+  it('does not render a link when disabled', () => {
     const wrapper = shallow(
       <ProgressTableLevelBubble {...defaultProps} disabled={true} />
     );
-    assert(wrapper.is('div'));
+    expect(wrapper.find(subComps.LinkWrapper)).to.have.lengthOf(0);
```

Comment:
@maureensturgeon re. your comment above about the wrapper -- imo this is a big enough improvement for this test to warrant it being its own subcomponent. it allows for the possibility of the link implementation changing without breaking this test. more to the point though, i'm just not a fan of the `is('div')` check because i don't think that's necessarily telling us what it purports to be verifying.

it's also not hard for me to imagine `LinkWrapper` evolving into something that we'd want to reuse (e.g. built-in analytics or url validation or something).

unless you think it makes the code harder to read, i'm in favor of keeping it. but lmk if you disagree.

Reply:
hmm, i guess i could just do a `find('a')` and verify the length is either 0 or 1. that would satisfy me of you want me to remove `LinkWrapper`.

Reply:
I could also imagine it evolving, so that makes sense to me. Would it be worth making a `LinkWrapper` component now and putting it somewhere common so we can start using it?